README.md

```diff
@@ -604,6 +604,113 @@ print(c.namespace.add(a=10, b=10))
 print(c(version='v2', prtocol='pickle').namespace.add(a=10, b=10))
 ```
 
+### Authentication
+KwikAPI supports `Basic` and `Bearer` authentication.
+
+**Basic authentication:**
+
+Setting up authentication at server side
+
+```python
+import tornado.ioloop
+import tornado.web
+
+from kwikapi.tornado import RequestHandler
+from kwikapi import API, Request, BasicServerAuthenticator
+
+class Calc(object):
+    def add(self, req: Request, a: int, b: int) -> int:
+        if not req.auth.is_authenticated:
+            raise Exception("No auth") # Or your logic
+        return a + b
+
+user_store = dict(username=dict(password='password'))
+auth = BasicServerAuthenticator(user_store=user_store)
+api = API(auth=auth)
+api.register(Calc(), 'v1')
+
+def make_app():
+    return tornado.web.Application([
+        (r'^/api/.*', RequestHandler, dict(api=api)),
+    ])  
+if __name__ == "__main__":
+    app = make_app()
+    app.listen(8818)
+    tornado.ioloop.IOLoop.current().start()
+```
+
+Using authentication at client side.
+```python
+from kwikapi import Client, BasicClientAuthenticator
+
+auth = BasicClientAuthenticator(username='username', password='password')
```

Comment:
@RamanjaneyuluIdavalapati instead of username='username', please use username='johndoe'